deltachat-ffi/src/lib.rs

```diff
@@ -1144,7 +1127,41 @@ pub unsafe extern "C" fn dc_delete_chat(context: *mut dc_context_t, chat_id: u32
             .delete(&ctx)
             .await
             .log_err(ctx, "Failed chat delete")
-            .unwrap_or(())
+            .ok();
+    })
+}
+
+#[no_mangle]
+pub unsafe extern "C" fn dc_block_chat(context: *mut dc_context_t, chat_id: u32) {
```

Comment:
you description says this only works for mailing lists.  that means you can call it on a chat where it won't work.  should you not return a success/fail value from this function in that case?

Reply:
I did the same as `dc_delete_chat`: log the error and do nothing.